### ISSUE TITLE
nosetests.json in different location than project root

### ISSUE THREAD

**Issue by bryan-lott** [thschenk/atom-nosetests#1]
It'd be great to be able to specify the location of the nosetests.json per project, the root of my python scripts isn't the same as my project root as it's a polyglot project.
Thanks!

**Comment by thschenk** [comment]
Would it work for you if python-nosetests would start searching for a nosetests.json in the same folder as the active file and move up directories as long as it cannot find one? I don't want to introduce project specific settings if that's not absolutely needed.

**Comment by bryan-lott** [comment]
I like that solution quite a bit better.  What would the behavior be if there were multiple .json files... i.e.:

```
project-root
    project1folder
        project1.json
    project2folder
        project2.json
```

could you run nosetests from each folder and display the combined results in the same pane?

**Comment by thschenk** [comment]
If you would use multiple nosetests.json files, which tests are executed would depend on the active file when you hit `F5`.

```
project-root/
    project1folder/
        subfolder/
            module1.py
            test_module1.py
        nosetests.json       (A)

    project2folder/
        module2.py
        test_module2.py
        nosetests.json       (B)

    javascript_project/
        main.js
```

If you have `module1.py` or `test_module1.py` opened, it would find nosetests.json (A). However, for `module2.py` or `test_module2.py` it would find nosetests.json (B).

When e.g. `main.js` is selected, there is no `nosetests.json` in any parent folder, so it would not be possible to run any tests.

Running tests from multiple projects at once is already possible if you put the `nosetests.json` in your project root:

```
project-root/

    nosetests.json

    project1folder/
        subfolder/
            module1.py
            test_module1.py

    project2folder/
        module2.py
        test_module2.py
        nosetests.json

    javascript_project/
        main.js
```

If you generate the initial `nosetests.json` with the command below, you would already be testing everything in your project folders:

```
nosetests --with-json-extended project1folder project2folder
```

**Comment by bryan-lott** [comment]
That sounds great.  It seems like the "average" user wouldn't be surprised by that behavior and it would definitely solve my use-case.  Thanks!!

**Comment by thschenk** [comment]
The new behaviour is implemented in v0.2.0.

**Comment by bryan-lott** [comment]
Just wanted to confirm this does exactly what I needed.  Thank you!!
